test/e2e/validation_test.go

```diff
@@ -464,6 +465,95 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testServicesConfigMap tests multiple aspects of expected functionality for the services ConfigMap
+// 1. It exists on operator startup 2. It is re-created when deleted 3. It is recreated if invalid contents are detected
+func testServicesConfigMap(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+
+	operatorVersion, err := getWMCOVersion()
+	require.NoError(t, err)
+	// There is no easy way to populate ldflags when running e2e tests without having to maintain the WMCO version in
+	// another location, so that version.Get() will return something other than "". Because of this,
+	// we cannot use servicescm.Name and must build to the services CM name by retrieving the version in another manner.
+	servicesConfigMapName := servicescm.NamePrefix + operatorVersion
+
+	// Ensure the windows-services ConfigMap exists in the cluster
+	_, err = tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(),
+		servicesConfigMapName, meta.GetOptions{})
+	require.NoErrorf(t, err, "error retrieving ConfigMap: %s", servicesConfigMapName)
+
+	// Delete and ensure re-creation
+	err = tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Delete(context.TODO(), servicesConfigMapName,
+		meta.DeleteOptions{})
+	require.NoErrorf(t, err, "error deleting ConfigMap: %s", servicesConfigMapName)
+	_, err = tc.waitForConfigMap(servicesConfigMapName)
+	require.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
```

Comment:
For clarity this can be converted in to a subtest i.e. `testServicesCMRecreation()`